src/partials/proposal.html

```diff
@@ -0,0 +1,58 @@
+<section class="section section__proposal">
+  <div class="container container__proposal">
```

Comment:
Please don't use container__proposal.
we hav classes .container and .container__content